packages/manager/apps/dedicated/client/app/billing/payment/method/add/component.js

```diff
@@ -8,11 +8,13 @@ export default {
   bindings: {
     addSteps: '<',
     currentUser: '<',
+    defaultPaymentMethodIntegration: '<',
     isLastStep: '<',
     isStepVisible: '<',
     getBackButtonHref: '<',
     model: '<',
     onPaymentMethodAdded: '<',
     paymentMethods: '<',
+    user: '<',
```

Comment:
Not able to use `Environment.getUser()` ?

Reply:
Used here : https://github.com/ovh/manager/pull/4368/files#diff-3e162df57e75b7270ca353f214694595c11b5c8dbcd8798293d7efb7e3c85ce4R136